packages/editor/src/validators/document/index.test.ts

```diff
@@ -81,4 +83,73 @@ describe('Document validations', () => {
       },
     });
   });
+
+  it('returns errors for potential lists', () => {
+    const editor = createTestEditor(`
+      <h3>Enkele paragraaf test</h3>
+          <p>- Lijst item<br />- Lijst item<br />- Lijst item</p>
+          <h3>Correct lijst</h3>
+          <ul>
+            <li>Test</li>
+            <li>Test</li>
+            <li>Test</li>
+          </ul>
+          <h3>Incorrect genummerde lijst</h3>
+          <p>1 - Test<br />2 - Test<br />3 - Test</p>
+          <p>1. Test<br>2. Test<br>3. Test</p>
+          <h3>Correcte genummerde lijst</h3>
+          <ol>
+            <li>Test</li>
+            <li>Test</li>
+            <li>Test</li>
+          </ol>
+          <h3>Losse paragrafen, waarschijnlijk een ongenummerde lijst</h3>
+          <p>- Losse paragrafen test</p>
+          <p>- Losse paragrafen test</p>
+          <p>- Losse paragrafen test<br />- Losse paragrafen test</p>
+          <h3>test</h3>
+          <p>* Losse paragrafen test<br />* Losse paragrafen test<br/>* Losse paragrafen test</p>
+          <h3>Losse paragrafen, waarschijnlijk een genummerde lijst</h3>
+          <p>1 - Losse paragrafen test</p>
+          <p>2 - Losse paragrafen test</p>
+          <p>3 - Losse paragrafen test</p>
+    `);
+
+    const result = documentMustHaveSemanticLists(editor);
+    expect(result).toEqual([
+      {
+        boundingBox: {
+          height: 0,
+          top: 0,
+        },
+        pos: 279,
+        severity: 'info',
+        tipPayload: {
+          prefix: '-',
+        },
+      },
+      {
+        boundingBox: {
+          height: 0,
+          top: 0,
+        },
+        pos: 304,
+        severity: 'info',
+        tipPayload: {
+          prefix: '-',
+        },
+      },
+      {
+        boundingBox: {
+          height: 0,
+          top: 0,
+        },
+        pos: 512,
+        severity: 'info',
+        tipPayload: {
+          prefix: '1',
+        },
+      },
+    ]);
+  });
```

Comment:
Perhaps it is wise make the test(s) verify the intended reporting:
* expect to get a list of errors
* expect a severity level for every error
* expect a bounding box for every error